### ISSUE TITLE
Specifying integration limits

### ISSUE THREAD

**Issue by Mulliken** [N3PDF/vegasflow#80]
Hi vegasflow developers!

Can I specify integration (upper and lower) limits in vegasflow? I searched the documentation but didn't found a way.

**Comment by scarlehoff** [comment]
Hi @Mulliken, thanks for opening the issue!

You cannot. But you can easily transform any 0-1 random variable to any x_min, x_max in your integrand, assuming `r` is the random variable between 0 and 1 and `x_min`, `x_max` your integration limits for this variable:

```python
x = x_min + r*(x_max - x_min)
jac = (x_max - x_min)
[...]
result *= jac
```

Since you changed the integration limits the result needs to be multiply the result by the jacobian of the transformation.


tbh, this is something that has come up a few times already in the past so I think we will either add the option or document what I wrote above (which might be obvious but better explicit than implicit!)

**Comment by Mulliken** [comment]
Thanks @scarlehoff! Adding this explicitly to the doc would help a lot!

Although now I understand the integration limits, I still have some questions about the code in [/example/simgauss_tf.py](https://github.com/N3PDF/vegasflow/blob/master/examples/simgauss_tf.py). What does [this line](https://github.com/N3PDF/vegasflow/blob/fc4a5fd2eb1134cdca5f5c98648d5019bfc70113/examples/simgauss_tf.py#L28) `coef = tf.reduce_sum(tf.range(n100 + 1))`? What is `n100` defined for?

Thanks!

**Comment by scarlehoff** [comment]
That is just an example, in this case the 100 is used several times so I put it as a variable to give a bit more conplexity to the example.

**Comment by Mulliken** [comment]
How n100 is related to the Gaussian integrand? For example, I can understand `coef += tf.reduce_sum(tf.square((xarr - 1.0 / 2.0) / a), axis=1)` is the exponent in the Gaussian integrand, but what does n100 do?

**Comment by scarlehoff** [comment]
It's just a way of adding complexity to the integrand. Actually, the first and third line cancel each other:

```
    coef = tf.reduce_sum(tf.range(n100 + 1))
    coef += tf.reduce_sum(tf.square((xarr - 1.0 / 2.0) / a), axis=1)
    coef -= (n100 + 1) * n100 / 2.0
```

so the result corresponds only to the line in the middle.

**Comment by Mulliken** [comment]
Gotcha! Thanks @scarlehoff !
